media-api/app/lib/UsageStore.scala

```diff
@@ -141,18 +141,18 @@ class UsageStore(
     case (_, status) if status.exceeded => status.usage.agency
   }.toList
 
-  def update() {
-    lastUpdated.send(_ => DateTime.now())
-    fetchUsage.foreach { usage => store.send(usage) }
+  def update(): Unit = {
+    store.send(fetchUsage)
   }
 
-  private def fetchUsage: Future[Map[String, UsageStatus]] = {
+  private def fetchUsage: Map[String, UsageStatus] = {
```

Comment:
this didn't need to be a future at all - the only futures it called were both wrappers around `Future.successful` - so unwrap the futures and just use synchronous code :)